packages/cashier/src/stores/payment-agent-store.ts

```diff
@@ -93,29 +108,28 @@ export default class PaymentAgentStore {
     }
 
     @action.bound
-    setList(pa_list) {
+    setList(pa_list: TPartialPaymentAgentList): void {
         this.list.push(pa_list);
     }
 
     @action.bound
-    clearList() {
-        this.list = [];
+    clearList(): void {
+        this.list = [] as unknown as IObservableArray<TPartialPaymentAgentList>;
     }
 
     @action.bound
-    async setPaymentAgentList(pa_list) {
-        const payment_agent_list = pa_list || (await this.getPaymentAgentList());
+    async setPaymentAgentList(): Promise<void> {
+        const payment_agent_list = await this.getPaymentAgentList();
         this.clearList();
-        this.clearSuppertedBanks();
-        // TODO: Once telephone, url and supported_banks removed from paymentagent_list.list we can remove them and just use the plural ones
+        this.clearSupportedBanks();
         try {
             payment_agent_list.paymentagent_list?.list.forEach(payment_agent => {
                 this.setList({
                     email: payment_agent.email,
-                    phones: payment_agent?.phone_numbers || payment_agent?.telephone,
+                    phones: payment_agent?.phone_numbers,
                     name: payment_agent.name,
                     supported_banks: payment_agent?.supported_payment_methods,
-                    urls: payment_agent?.urls || payment_agent?.url,
+                    urls: payment_agent?.urls,
```

Comment:
remove `payment_agent?.url` because we don't use it now (BE rename this field from `url` to `urls`) and his value is always an array (e.g `urls : [{url : '1234567']}`)